process/css-styleguide/input/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input</title>
+    <link rel="stylesheet" href="../heading/heading.css" />
+    <link rel="stylesheet" href="./input.css" />
+    <link rel="stylesheet" href="../spacing.css" />
+    <style>
+      .input-container--local {
+        padding: 1rem;
+      }
+    </style>
+  </head>
+  <body class="p-2">
+    <div class="input-container--local">
+      <input type="text" class="input mr-3 mb-4" placeholder="Write something" />
+      <div><label class="typography poppins body-s">Label</label></div>
+      <div>
+        <input type="text" class="input mb-4" placeholder="Write something" />
+      </div>
+      <span class="typography poppins body-xs input--message">Lorem Message</span>

```

Comment:
`Lorem Message` and `Label` appear in the same line in the rendered HTML:
![Capture d’écran 2022-07-28 à 09 58 22](https://user-images.githubusercontent.com/48251481/181521263-70777eee-50e6-43ba-8852-21199e959beb.png)